api_docs/changelog.md

```diff
@@ -20,6 +20,19 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 323**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
+  [`GET /streams`](/api/get-streams),
+  [`GET /streams/{stream_id}`](/api/get-stream-by-id): Added a new
+  field `is_recently_active` to stream, which stores if a message
+  has been sent to the stream recently. This is used by the web app
+  to render streams in the left sidebar in correct order during initial
+  loading of the app. This new field will help replace the previous method
+  of using locally available messages to determine the active status of
+  streams in the left sidebar which leads to a lot of jumping around of
+  streams after initial app load.
```

Comment:
We try to keep the notes in this file concise, just explaining the change and its relevance to clients, without explaining the details. This file is huge even in brief mode :).

``` diff
diff --git a/api_docs/changelog.md b/api_docs/changelog.md
index 01d219485b..4f3b0fb03d 100644
--- a/api_docs/changelog.md
+++ b/api_docs/changelog.md
@@ -22,16 +22,11 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 323**
 
-* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
-  [`GET /streams`](/api/get-streams),
+* [`POST /register`](/api/register-queue), [`GET
+  /events`](/api/get-events), [`GET /streams`](/api/get-streams),
   [`GET /streams/{stream_id}`](/api/get-stream-by-id): Added a new
-  field `is_recently_active` to stream, which stores if a message
-  has been sent to the stream recently. This is used by the web app
-  to render streams in the left sidebar in correct order during initial
-  loading of the app. This new field will help replace the previous method
-  of using locally available messages to determine the active status of
-  streams in the left sidebar which leads to a lot of jumping around of
-  streams after initial app load.
+  field `is_recently_active` to stream objects as a new deterministic
+  source of truth for `demote_inactive_streams` activity decisions.
 
 **Feature level 322**
 
```